src/containers/Charts/Chart.js

```diff
@@ -9,6 +9,8 @@ const Chart = ({ solarData }) => {
 
   const [chartType, updateChartType] = useState('bar')
   const [yAxisLabel, updateYAxisLabel] = useState('(kWh / m^2) / day')
+  const pieColors = ['#a93ad7', '#077ed6', '#96d9ff', '#5fb7d4', '#7cdddd', '#26d7ae', '#2dcb75', '#1caa2f', '#d5f30d', '#feec01', '#fba052', '#ff1d00']
```

Comment:
Rainbow!

Reply:
Great job with conditional function for rendering multiple colors on render of the pie chart!